apps/cookbook/src/app/examples/slides-example/example-card-content.ts

```diff
@@ -39,9 +39,9 @@ export class CookbookExampleCardContentComponent implements OnInit, OnChanges {
     this.cardContent = this.data.cardContent;
   }
 
-  ngOnChanges(changes: SimpleChanges) {
-    // not in use in this example - but gives you access to the index of any slide from the component loaded inside the slide
-    let activeSlide = changes.activeSlideIndex.currentValue || 0;
-    // doSomethingWithTheActiveSlideIndex(activeSlide);
-  }
+  // ngOnChanges(changes: SimpleChanges) {
+  //   not in use in this example - but gives you access to the index of any slide from the component loaded inside the slide
+  //   let activeSlide = changes.activeSlideIndex.currentValue || 0;
+  //   doSomethingWithTheActiveSlideIndex(activeSlide);
+  // }
```

Comment:
Consider removing completely.

Reply:
Yeah, I was bit unsure what to do about it - I'll nuke it

Reply:
Maybe the right thing to do is to make it a code-snippet in slides in the cookbook? Thoughts?

Reply:
Opted to remove it - I dont think it belongs in the cookbook really. It's more of an Angular thing